addons/mail/static/tests/discuss/core/discuss.test.js

```diff
@@ -43,18 +43,24 @@ test("bus subscription is refreshed when channel is joined", async () => {
         `${later.year}-${later.month}-${later.day} ${later.hour}:${later.minute}:${later.second}`
     );
     const env = await start();
-    const imStatusChannels = [];
+    const expectedSubscribes = [];
     for (const { type, id } of env.services["mail.store"].imStatusTrackedPersonas) {
```

Comment:
Previous version changed code to use `getService` instead of `env`, shouldn't we do the same here?

Reply:
yeah, will do